lib/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import { doHighlight, deserializeHighlights, serializeHighlights, removeHighlights, createWrapper, highlightRange } from "../src/Library";
+import { doHighlight, deserializeHighlights, serializeHighlights, removeHighlights, createWrapper, highlightRange } from "./Library";
```

Comment:
these paths are generated while running the `npm run build` cmd.  While these could be correct, the correct answer would be to update the build process to replace these values

Reply:
I've updated the build process to handle these. if anything comes up, file an issue